File: java/programmers/1단계/src/자연수뒤집어배열로만들기.java
import java.util.Arrays;

public class 자연수뒤집어배열로만들기 {
    public int[] solution(long n) {
        int[] answer = new int[String.valueOf(n).length()];
        int num = 0;
        while(n > 0){
            answer[num++] = (int)(n % 10);
            n = n / 10;
        }
        return answer;
    }
    public static void main(String[] args) {
        자연수뒤집어배열로만들기 sol = new 자연수뒤집어배열로만들기();
        long n = 12345;
        System.out.println(Arrays.toString(sol.solution(n)));
    }
}

File: python/python_Programmers/1단계/자연수뒤집어배열로 만들기.py
# 내풀이
def solution(n):
    answer = list(map(int,str(n)))
    answer.reverse()
    return answer

# 모범 풀이 [::-1]은 거꾸로 인덱싱
def digit_reverse(n):
    return [int(i) for i in str(n)][::-1]

n = 12345
print(solution(n))
print(digit_reverse(n))